lib/src/main/java/uk/gov/di/ipv/cri/kbv/api/gateway/KBVGatewayFactory.java

```diff
@@ -13,40 +13,27 @@
 import uk.gov.di.ipv.cri.kbv.api.service.MetricsService;
 
 public class KBVGatewayFactory {
-    public static final String IIQ_DATABASE_MODE_PARAM_NAME = "IIQDatabaseMode";
-    private final KBVGateway kbvGateway;
 
-    public KBVGatewayFactory(ConfigurationService configurationService) {
-        HeaderHandler headerHandler =
-                new HeaderHandler(
-                        new Base64TokenCacheLoader(
-                                new SoapToken(
-                                        "GDS DI",
-                                        true,
-                                        new TokenService(),
-                                        configurationService.getSecretValue(
-                                                "experian/iiq-wasp-service"))));
-
-        new KeyStoreLoader(configurationService).load();
```

Comment:
Is there a specific reason the KeyStoreLoader invocation has been moved out of this class?